index.html

```diff
@@ -1021,14 +1021,14 @@ <h2>PNG image</h2>
       <!-- <ol start="1"> -->
 
       <dl>
-        <dt><dfn>Truecolour with alpha</dfn></dt>
+        <dt><dfn>Truecolor with alpha</dfn></dt>
         <dd>Each <a>pixel</a> consists of four <a>samples</a>: red, green, blue and <a>alpha</a>.</dd>
 
         <dt><dfn>Greyscale with alpha</dfn></dt>
         <dd>Each <a>pixel</a> consists of two <a>samples</a>: a <a>grey sample</a> and an <a>alpha</a> sample.</dd>
 
         <!-- Maintain a fragment named "3truecolour" to preserve incoming links to it -->
-        <dt><dfn id="3truecolour">Truecolour</dfn></dt>
+        <dt id="truecolor"><dfn id="3truecolour">Truecolor</dfn></dt>
```

Comment:
I would either not add an ID or give the `<dfn>` the improved ID.

Reply:
We want to preserve inbound link fragments where possible, to keep compatibility from the 2003 second edition. That is why this one has both; it enables ReSpec `<a>truecolor</a>` to work while also preserving the older id as well.

Reply:
ReSpec doesn't key off the `dfn`? Weird but okay.